### ISSUE TITLE
Incorporate sounds

### ISSUE THREAD

**Issue by ericthompson** [grokpot/Routy#39]
So the sounds are recorded - lets get some feedback going.

We have an introduction sound that just says "Routy" in a sort of high-pitched slightly roboty voice.

We have an error noise that sounds like a sad robot.

And finally we have a clicking noise that I made. With my mouth. Yes.

Want to use them?

**Comment by grokpot** [comment]
Yeah I'm definitely game to try them out.

**Comment by ericthompson** [comment]
So I tried to go through and use SoundPool to play the sounds but I'm a little confused on when/where are the proper places for it to release the resources it uses. It's simple, in practice, but I end up breaking a lot because I'm not 100% sure where to release/nullify.

Thoughts?

**Comment by jtran19** [comment]
I think the general practice (from what I gathered during the sound tutorial) is to obtain the resources in onResume and release them in onPause.

**Comment by ericthompson** [comment]
So there are a couple of things that kept going wrong - in our example from class, we had only one activity, but here we have multiple. This means I ran into problems when the user goes "back" into a previous screen. Then we would have a nulled out soundpool object trying to be re-referenced which would crash the app. I tried to rebuild it there as well, but it does something different and I would end up with another soundpool object that has to be dealt with.

Unfortunately, I don't see another way around using sounds than to build a new soundpool object in each activity, but this is a pain. Any ideas?

**Comment by jtran19** [comment]
Unless Google (search) says differently, it seems like "one SoundPool per Activity" is what they intended.  I don't know of a way to create a SoundPool and hang on to it for the life of the app (versus life of an Activity).  Since you can essentially "quit" the app at any screen (by hitting the Home softkey) it would make sense that there is one SoundPool per Activity.  Then it can be obtained/released for the life of Activity.

**Comment by ericthompson** [comment]
OK - well then I'll give it a go again later. I just kept having it crash after a few hours of me tinkering with one of the activities - Origin. No matter what I tried to do, I didn't always seem to catch all of the functioning of it and it would eventually die. I closed it, came back; moved to the next screen and went back, and sometimes it died while in the activity itself. I just ended up having a bunch of trouble.
